app/assets/stylesheets/global.scss

```diff
@@ -0,0 +1,7 @@
+.top-bar .plain-text {
+  font-family: "Helvetica Neue", Helvetica, Roboto, Arial, Sans-Serif;
```

Comment:
Properties should be ordered color, font-family, font-size, line-height, padding